src/app/labeling/qgsmaptoollabel.cpp

```diff
@@ -1054,6 +1189,62 @@ bool QgsMapToolLabel::labelMoveable( QgsVectorLayer *vlayer, const QgsPalLayerSe
   return false;
 }
 
+bool QgsMapToolLabel::labelAnchorPercentMovable( QgsVectorLayer *vlayer, const QgsPalLayerSettings &settings, int &lineAnchorPercentCol, int &lineAnchorClippingCol,  int &lineAnchorTypeCol,  int &lineAnchorTextPointCol ) const
+{
+  lineAnchorPercentCol = -1;
+  lineAnchorClippingCol = -1;
+  lineAnchorTypeCol = -1;
+  lineAnchorTextPointCol = -1;
+
+  bool ret { true };
```

Comment:
as far as I can set, the ret variable could be removed, as it is intialized here to true, and re-assigned to true only, so just a final "return true" would do.

Reply:
I think my intention was to return `false` in case no DD properties is active, I'll change the initialization to `false`.

Reply:
Ok, I refactored the whole method.